src/components/resource-side-panel/ResourceSidePanel.tsx

```diff
@@ -74,6 +80,27 @@ export function ResourceSidePanel({
   // AutoRAG job polling hook
   const { jobStatus, startPolling } = useAutoRAGPolling();
 
+  const handleCreateCampaign = async () => {
+    if (!campaignName.trim()) return;
+
+    try {
+      // TODO: Implement actual campaign creation API call
```

Comment:
You leaving this for me? idk if there's a campaign dao already for sure, but I think there is